scripts/hetnet_id_extractor.py

```diff
@@ -0,0 +1,63 @@
+import argparse
+
+import pandas as pd
+
+
+def filter_tags(infile, outfile):
+    extracted_tag_df = pd.read_csv(infile, sep="\t")
+    hetnet_chemical_df = load_chemical_df()
+    hetnet_disease_df = load_disease_df()
+    hetnet_gene_df = load_gene_df()
+
+    # Covert chemical IDs
+    chemical_merged_df = pd.merge(extracted_tag_df, hetnet_chemical_df[["drugbank_id", "identifier"]], left_on="identifier", right_on="identifier").drop_duplicates()
+    chemical_merged_df["type"] = "Compound"
+    chemical_merged_df = chemical_merged_df[["pubmed_id", "type", "offset", "end", "identifier"]]
+
+    # Convert Disease IDs
+    disease_merged_df = pd.merge(extracted_tag_df, hetnet_disease_df[["doid_code", "resource_id"]], left_on="identifier", right_on="resource_id").drop_duplicates()
+    disease_merged_df = disease_merged_df[["pubmed_id", "type", "offset", "end", "doid_code"]]
+
+    # Verify Gene IDs are human genes
+    gene_df = extracted_tag_df[extracted_tag_df["type"] == "Gene"]
+    gene_final_df = gene_df[gene_df["identifier"].isin(hetnet_gene_df["GeneID"])]
+
+    final_df = gene_final_df
+    final_df = final_df.append(chemical_merged_df)
+    final_df = final_df.append(disease_merged_df)
+
+    final_df[["pubmed_id", "type", "identifier", "offset", "end"]].to_csv(args.output, sep="\t", index=False)
+
+
+def load_disease_df():
+    """
+    Load the mesh-hetnet id mapping table.
+    Return a dataframe of id mappings
+    """
+    url = 'https://raw.githubusercontent.com/dhimmel/disease-ontology/052ffcc960f5897a0575f5feff904ca84b7d2c1d/data/xrefs-prop-slim.tsv'
```

Comment:
Do we want to filter this dataframe to just the relevant types for pubtator here?

Also can this file be untracked from this repo, if we're reading from URL?